go.mod

```diff
@@ -82,15 +82,15 @@ require (
 	github.com/libp2p/go-libp2p-tls v0.1.3
 	github.com/libp2p/go-libp2p-yamux v0.5.3
 	github.com/libp2p/go-socket-activation v0.0.2
-	github.com/libp2p/go-tcp-transport v0.2.1
+	github.com/libp2p/go-tcp-transport v0.2.2
 	github.com/libp2p/go-ws-transport v0.4.0
 	github.com/lucas-clemente/quic-go v0.19.3
 	github.com/miekg/dns v1.1.41
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/multiformats/go-multiaddr v0.3.1
 	github.com/multiformats/go-multiaddr-dns v0.3.1
 	github.com/multiformats/go-multibase v0.0.3
-	github.com/multiformats/go-multihash v0.0.14
+	github.com/multiformats/go-multihash v0.0.15
```

Comment:
Don't update go-multihash in this PR since it'll cause a sharness test failure (being solved in another PR). It also doesn't seem needed here.

Reply:
I didn't do this intentionally, `go mod` did this. It's because of this PR: https://github.com/libp2p/go-tcp-transport/pull/75/files. I don't really understand why this PR pulled in go-multihash though.